src/arch/xtensa/boot_loader.c

```diff
@@ -168,9 +168,11 @@ static void parse_manifest(void)
 }
 #endif
 
-/* power off unused HPSRAM */
-#if CONFIG_CANNONLAKE
-
+#if CONFIG_CAVS_VERSION_1_8 || CONFIG_CAVS_VERSION_2_0 || \
```

Comment:
Maybe "#if CAVS_VERSION >= CAVS_VERSION_1_8" is valid too?

Reply:
ack